Assets/Scripts/Entity/AttackLogic.cs

```diff
@@ -135,9 +178,11 @@ private void HitFinished()
             if (Status == AttackStatus.Hit)
             {
                 // deal damage
-                // Add damage boost from weapon
-                int boostedDamage = damage + InventoryManager.Instance.DamageBoostFromEquipable;
```

Comment:
Lol wtf 😂 Was this applied to every entity?

Reply:
Who would add something like that... He must be very smart 😅